### ISSUE TITLE
ignoreRow: [] issue

### ISSUE THREAD

**Issue by Ranjith080216** [kayalshri/tableExport.jquery.plugin#113]
Hi,

I wanted to remove one particular row and column from the excel downloaded, after going through several documentations i found that i can use ignoreRow: [] and ignoreColumn: [] fields inside tableExport({ });

ignoreColumn: [] is working fine i can remove which ever column i want to by passing index but where as i am not to remove row by using ignoreRow:[].

Please find the attached code for reference, can someone help me to solve this issue?

<!DOCTYPE html>
<html>

<head>
  <meta charset="UTF-8">
  <title>HTML table Export</title>

    
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js" defer type="text/javascript"></script>
    <script src="libs/js-xlsx/xlsx.core.min.js" defer type="text/javascript"></script>
    <script src="libs/FileSaver/FileSaver.min.js" defer type="text/javascript"></script> 
    <script src="libs/tableExport.min.js" defer type="text/javascript"></script>
    
    <script type="text/javaScript">         
        var sFileName = 'ngophi';
        function ExportXLSX(){
            //alert("Hello....")
            $('#Event').tableExport({fileName: sFileName,
                        type: 'xlsx',
                        ignoreRow: [2],
                        ignoreColumn: [2]
                       });
        }
    </script>
 <style type="text/css">
     body {
        font-size: 12pt;
        font-family: Calibri;
        padding : 10px;
    }
    table {
        border: 1px solid black;
    }
    th {
        border: 1px solid black;
        padding: 5px;
        background-color:grey;
        color: white;
    }
    td {
        border: 1px solid black;
        padding: 5px;
    }
    input {
        font-size: 12pt;
        font-family: Calibri;
    }
 </style>
</head>
<body>  
<a href="#" onClick="ExportXLSX();">DownloadXLSX</a> 
<br/>
<br/>
<div id="Event">
    <table>
        <tr>
            <th>Column One</th>
            <th>Column Two</th>
            <th>Column Three</th>
        </tr>
        <tr>
            <td>row1 Col1</td>
            <td>row1 Col2</td>
            <td>row1 Col3</td>
        </tr>
        <tr>
            <td>row2 Col1</td>
            <td>row2 Col2</td>
            <td>row2 Col3</td>
        </tr>
        <tr>
            <td>row3 Col1</td>
            <td>row3 Col2</td>
            <td><a href="http://www.jquery2dotnet.com/">http://www.jquery2dotnet.com/</a>
            </td>
        </tr>
    </table>
</div>
</body>
</html>

**Comment by hhurz** [comment]
FYI: I think you opened this issue on the wrong project site. It seems that you are using a [fork](https://github.com/hhurz/tableExport.jquery.plugin) of kayalshri's plugin. I'm the author of that fork 😉 So please close the issue here and create it [there](https://github.com/hhurz/tableExport.jquery.plugin/issues). Btw I confirm that there is a problem with the ignoreRow option when exporting to xlsx.

**Comment by Ranjith080216** [comment]
Thanks for the reply, as suggested closing the issue here and opened one in the other repo.
